zerver/tornado/event_queue.py

```diff
@@ -201,7 +201,7 @@ def finish_current_handler(self) -> bool:
                 logging.exception(err_msg, stack_info=True)
             finally:
                 self.disconnect_handler()
-            return True 
+            return True
```

Comment:
This time you have a spurious whitespace fix in the second commit that belongs in the first commit.

(Are you recreating your branch from scratch each time? Perhaps you weren’t aware that Git provides tools for this? Our [commit discipline](https://zulip.readthedocs.io/en/latest/contributing/version-control.html#commit-discipline) documentation links to our cheat sheet on [fixing commits](https://zulip.readthedocs.io/en/latest/git/fixing-commits.html), and there’s more documentation in the Git manual: [cleaning up history](https://git-scm.com/docs/user-manual#cleaning-up-history), [interactive rebases](https://git-scm.com/docs/user-manual#interactive-rebase).)

Reply:
> This time you have a spurious whitespace fix in the second commit that belongs in the first commit.
> 
> (Are you recreating your branch from scratch each time? Perhaps you weren’t aware that Git provides tools for this? Our [commit discipline](https://zulip.readthedocs.io/en/latest/contributing/version-control.html#commit-discipline) documentation links to our cheat sheet on [fixing commits](https://zulip.readthedocs.io/en/latest/git/fixing-commits.html), and there’s more documentation in the Git manual: [cleaning up history](https://git-scm.com/docs/user-manual#cleaning-up-history), [interactive rebases](https://git-scm.com/docs/user-manual#interactive-rebase).)

Thank you so much for the refereces! I am still trying to learn git!